docs/static/jvm.asciidoc

```diff
@@ -50,7 +50,8 @@ Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11.0.1+13-LTS, mixed mode)
 ==== `LS_JAVA_HOME`
 
 {ls} uses the Java version set in `LS_JAVA_HOME`. The `LS_JAVA_HOME` environment
-variable must be set for {ls} to operate correctly.
+variable must be set for {ls} to operate correctly if you prefer to use custom JDK 
```

Comment:
I would rewrite as:
```
If you prefer to use custom JDK other than the version that is bundled, then the `LS_JAVA_HOME` environment
variable must be set.
```

Reply:
cc @karenzone

Reply:
@andsel I updated the sentence as your suggestion

Reply:
I would ask  @karenzone for a quick eye